guides/v2.3/magento-functional-testing-framework/2.3/getting-started.md

```diff
@@ -137,101 +140,97 @@ Example: `MAGENTO_ADMIN_USERNAME=admin`
-Only change or specify `SELENIUM_*` values if you are not running Selenium locally, or if you have changed your Selenium Server configuration.
+In the `magento2/dev/tests/acceptance` directory, run the following command to enable the MFTF to send Magento CLI commands to your Magento instance.
 
-Your environment settings form the path to your running Selenium Server.
-Example:
-```
-http://127.0.0.1:4444/wd/hub
+ ```bash
+cp dev/tests/acceptance/.htaccess.sample dev/tests/acceptance/.htaccess
 ```
 
-### Step 5. (Optional) Copy `command.php` into Magento installation {#add-cli-commands}
+#### (Optional) Copy `command.php` into Magento installation {#add-cli-commands}
 
-If you are installing the MFTF not from your Magento installation, locate the `command.php` file in the MFTF:
+If you are installing the MFTF from the other location than your Magento installation, point to the `command.php` file in the MFTF source code:
 
 `magento2-functional-testing-framework/etc/config/command.php`
 
-And copy it into your Magento installation under: 
+And copy it into your Magento installation: 
 
-`magento2/dev/tests/acceptance/utils/command.php`
+`magento2/dev/tests/acceptance/utils/`
 
-If you are installing the MFTF from inside your Magento installation, this is automatically done when you [build the project].
+If you are installing the MFTF from inside your Magento installation, this is automatically done when you [build the project][].
 
 {:.bs-callout .bs-callout-warning}
 If you do not have access to your Magento installation and cannot complete the step you will not be able to use the Magento CLI commands.
 
-### Step 6. Enable the Magento CLI commands
+### Step 7. Generate and run tests   {#run-tests}
 
-In the `magento2/dev/tests/acceptance` directory, run the following command to enable the MFTF to send Magento CLI commands to your Magento instance.
+To run tests, you need a running Selenium server and [`mftf`][] with required parameters.
 
- ```bash
-cp .htaccess.sample .htaccess
-```
-
-### Step 7. Generate existing tests     {#generate-tests}
+#### Run the Selenium server    {#selenium-server}
 
-In the `magento2/` root directory, run the following command to generate tests as PHP classes from XML files:
+Run the Selenium server in terminal.
+Example of the run for Chrome:
 
 ```bash
-vendor/bin/mftf generate:tests
+cd <path_to_directory_with_selenium_server_and_webdiver>/
+```
+```bash
+java -Dwebdriver.chrome.driver=chromedriver -jar selenium-server-standalone-3.14.0.jar
 ```
 
-### Step 8. Run tests   {#run-tests}
+If you do not have the Selenium server yet:
 
-To run tests, you need a running Selenium server and a [`codecept`] or [`mftf`] with required parameters.
+1. [Download the latest Selenium Server][].
 
-#### Run the Selenium server    {#selenium-server}
+2. [Download a Selenium web driver for your web browser][] into the same directory that contains the Selenium server.
 
-Run the Selenium server in terminal:
```

Comment:
The steps for setting up the Selenium server seem out of place here -- Seems like this info should be in the installation section, and then you can just point to that link here.   

Or at the top of the Getting Started, include a Prerequisites section and put the Selenium server info in that section.  It's not great to have to interrupt the flow of Getting Started to download and install required software.

Reply:
It's already in the prerequisites